package.json

```diff
@@ -12,7 +12,9 @@
     "eslint": "npm run lint-js",
     "lint-js": "eslint -c .eslintrc.js --ext .jsx,.js scripts",
     "lint-sass": "sass-lint -c .sass-lint.yml -v -q 'styles/**/*.scss'",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "deploy:staging": "./bin/deploy.sh staging",
```

Comment:
can't we do a single command, that accepts produtcion/staging as an argument?

Reply:
Actually I thought wrapping this would prevent running `./bin/deploy.sh monkey`. Since we only have 2 envs...